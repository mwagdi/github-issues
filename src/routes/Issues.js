import React,{ Component } from 'react';

class Issues extends Component{
    render(){
        return(
            <div className="issues__body">
                <h3>Issues</h3>
            </div>
        )
    }
}

export default Issues;