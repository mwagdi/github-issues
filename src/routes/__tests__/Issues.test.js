import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Issues from '../Issues';

test('Issues renders correctly',() => {
    const wrapper = shallow(<Issues />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})