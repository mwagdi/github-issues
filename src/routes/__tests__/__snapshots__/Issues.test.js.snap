// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issues renders correctly 1`] = `
<div
  className="issues__body"
>
  <h3>
    Issues
  </h3>
</div>
`;
